### PR TITLE
tests(init): assert for webpack config file content

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
test update

**Did you add tests for your changes?**
Updated `init` test suite.

**If relevant, did you update the documentation?**
N/A

**Summary**
Updated the `init` test suite to make sure that `webpack.config.js` is not empty.

**Does this PR introduce a breaking change?**
Nope

**Other information**
#2110 #2247

### CODE REVIEW

File: test/init/language/css/init-language-css.test.js [hunk 1]
```diff
@@ -32,12 +32,43 @@ describe('init with SCSS', () => {
...
 
         files.forEach((file) => {
             expect(fs.existsSync(resolve(genPath, file))).toBeTruthy();
         });
 
+        const webpackConfig = require(join(genPath, './webpack.config.js'));
+
+        expect(webpackConfig.module.rules).toEqual([
+            {
+                test: /.(scss|css)$/,
```
Comment (alexander-akait) [inline test/init/language/css/init-language-css.test.js:45]: Need fix it, it is wrong

**Review by rishabh3112 — APPROVED** [review]

**Review by alexander-akait — APPROVED** [review]
Should we close issue, they are fixed?

**Review by snitin315 — APPROVED** [review]

**Comment by alexander-akait** [comment]
@jamesgeorge007 Please answer the question

**Comment by jamesgeorge007** [comment]
Yup, it works fine, as evident from the test suite.

**Pull request merged**